### PR TITLE
Leaderboard: setup project

### PR DESCRIPTION
**Pull request by NayanKaran**
Hi,

I have implemented the following project requirements. Please verify.

### Project requirements

- **IMPORTANT NOTE** This is the first time you will use [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md). Keep that in mind!
- Create an **npm project with webpack**. (If you need a refresher, check the previous project or webpack exercise).
- Write plain HTML markup with minimum styling (just to make the layout), because you will implement styles in the following steps. Use this wireframe as a guide:

<p align="center">
  Basic UI
</p>

- Don't make API calls (you'll do it in a following lesson).
- Use ES6 modules, with *import* and *export*.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -3,9 +3,32 @@
...
+        <section id="add-score-section">
+            <h2>Add your score</h2>
+            <form id="score-form">
+                <input type="text" name="name" id="name" placeholder="Your name"><br><br>
+                <input type="number" name="score" id="score" placeholder="Your score"><br><br>
+                <input type="submit" value="Submit">
+            </form>
+        </section>
+    </main>
 </body>
```
Comment (Munsa1) [inline src/index.html:33]: - [x] Its required to make the wireframe similar to the provided design. Kindly address this issue.

- Your design
image

- Provided design
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @NayanKaran, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (NayanKaran) [reply src/index.html:33]: @Munsa1 Thank you for your feedback. I have already done the changes. https://nayankaran.github.io/Leaderboard/

**Review by OLIPLICHE — APPROVED** [review]
Hi @NayanKaran , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**